xml/Microsoft.VisualBasic.ApplicationServices/ConsoleApplicationBase.xml

```diff
@@ -85,7 +85,7 @@
       <Parameters />
       <Docs>
         <summary>Initializes a new instance of the <see cref="T:Microsoft.VisualBasic.ApplicationServices.ConsoleApplicationBase" /> class.</summary>
-        <remarks>To be added.</remarks>
+        <remarks>We have to have a parameterless constructor because the platform specific Application  object derives from this one and it doesn't define a constructor.  The partial class generated by the designer defines the constructor in order to configure the application.</remarks>
```

Comment:
This seems like an internal-only remark. If not, it should be reworded a bit to not start with "we have to...".

Reply:
Agreed, this seems like an internal remark. Consider removing the remark.

Reply:
```suggestion
        <remarks>To be added.</remarks>
```